CHANGELOG.md

```diff
@@ -9,7 +9,8 @@
 
 ## Unreleased
 
-* Add data attributes to attachment links using JS [(PR #3330)](https://github.com/alphagov/govuk_publishing_components/pull/3330)
+* Update GA4 index parameter on related navigation ([PR #3346](https://github.com/alphagov/govuk_publishing_components/pull/3346))
+* Add data attributes to attachment links using JS ([PR #3330](https://github.com/alphagov/govuk_publishing_components/pull/3330))
```

Comment:
Just noticed, this is in the wrong place, this was already released.

Reply:
Good spot, something weird happened there. Fixed now, thanks!